source/MaterialXCore/Util.cpp

```diff
@@ -6,6 +6,8 @@
 #include <MaterialXCore/Types.h>
 
 #include <cctype>
+#include <sstream>
```

Comment:
Are these new includes truly needed here?  I don't see any new usage of std::stringstream, for example, which would require the sstream header to be present.

Reply:
Not required will update